maintainers/maintainer-list.nix

```diff
@@ -4115,6 +4115,12 @@
     githubId = 22533224;
     name = "Emma Bastås";
   };
+  emattiza = {
```

Comment:
When I added myself to this file, I was asked to do it as a separate commit (with message `maintainers: add NAME`). To be in alphabetical order, the entry should go before `emmabastas`, instead of after.